bzt/jmx2yaml.py

```diff
@@ -135,7 +136,10 @@ def _get_concurrency(self, element):
         concurrency option in tg execution settings
         :return:
         """
-        concurrency = self._get_option_string_with_default(element, 'ThreadGroup.num_threads', "concurrency", 1)
+        if element.tag == "ThreadGroup":
+            concurrency = self._get_option_string_with_default(element, 'ThreadGroup.num_threads', "concurrency", 1)
+        else:
+            concurrency = self._get_option_string_with_default(element, 'TargetLevel', "concurrency", 1)
```

Comment:
It's better to set 'field_name' according to element.tag and set concurrency later isn't it?